filecoindot/src/ocw/de.rs

```diff
@@ -6,7 +6,7 @@ use serde::de::{Deserialize, Deserializer, Error as _};
 use serde_json::Value;
 
 /// deserialize json `Value` to `Vec<u8>`
-pub fn bytes<'de, D>(data: D) -> Result<Vec<u8>, D::Error>
+pub fn bytes<'de, D>(data: D) -> core::result::Result<Vec<u8>, D::Error>
```

Comment:
Is it necessary to use `core::result::Result` because `std` is not enabled by default.
There is also `Result` in [sp_std](https://paritytech.github.io/substrate/monthly-2021-09+1/sp_std/result/index.html). Though, I'm not sure which one is the correct Result enum is more appropriate.

Reply:
both `std::result::Result` and `sp_std::result::Result` are re-exported from `core::result::Result`

good catch, `core::result::Result` will make this more compact